dev/com.ibm.websphere.appserver.features/visibility/public/servlet/com.ibm.websphere.appserver.servlet.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.servlet
```

Comment:
Even this feature should be io.openliberty prefixed.  There are not tolerates needed for servlet so we don't have issues with the prefix of the versionless feature being different than the versioned features.